curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/logo.md

```diff
@@ -7,21 +7,18 @@ videoUrl:
 
 ## Description
 <section id='description'>
-Welcome to freeCodeCamp's HTML coding challenges. These will walk you through web development step-by-step.
-First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
-Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
-Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
-Opening tags look like this:
-<code>&#60;h1&#62;</code>
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+Sometimes we need to select any objects at the image. OpenCV has drawing functions. In this tutorial, we’ll look at features like cv.ellipse() and cv.circle(). The main aim of this lesson is show how use this function and draw openCV logo.
```

Comment:
`Sometimes we need to select any objects at the image`

What is object selection? I think it's OK to start with `OpenCV has drawing functions which let you draw simple geometric primitives.`